platform/backend/core/src/integrationTest/kotlin/io/hamal/core/req/handler/trigger/CreateTriggerHandlerTest.kt

```diff
@@ -92,7 +96,37 @@ internal class CreateTriggerHandlerTest : BaseReqHandlerTest() {
         }
 
         @Test
-        fun `Tries to create trigger, but func does not exists`() {
+        fun `Tries to create an invalid trigger`() {
```

Comment:
This test name does not describe the problem case you want to test - as in: What is an invalid trigger in this case